hw8/src/BoundedQueue.java

```diff
@@ -0,0 +1,65 @@
+import java.util.*;
+
+public class BoundedQueue <E> {
+
+    protected List<E> rep;
+    protected int size = 0;  // max size of queue
```

Comment:
Should we keep size? I'm thinking yes so that we know the max size of the Queue, but not sure if this would be the same thing as initializing the ArrayList to be size `size`

Reply:
yeah i think we should and to initialize the arrayList with size.